src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -0,0 +1,81 @@
+package org.carlmontrobotics.subsystems;
+
+import static org.carlmontrobotics.Constants.Limelight.*;
+import static org.carlmontrobotics.Constants.Limelight.Apriltag.*;
+
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.estimator.SwerveDrivePoseEstimator;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Limelight extends SubsystemBase {
+  private final Drivetrain drivetrain;
+  private final SwerveDrivePoseEstimator poseEstimator;
+  private double tv, tx;
+  private double[] botPose = null;
+  private double[] targetPose = null;
+  private Pose3d botpose;
```

Comment:
Please don't have `botPose` and `botpose` as variables, they can be very easily confused.